src/main/resources/anniv5Resources/localization/eng/royaltypack/Cardstrings.json

```diff
@@ -0,0 +1,47 @@
+{
+  "${ModID}:RetainerStrike": {
+    "NAME": "Retainer Strike",
+    "DESCRIPTION": "Deal !D! damage. NL Retain a random card this turn.",
+    "UPGRADE_DESCRIPTION": "Deal !D! damage. NL Retain a card this turn."
+  },
+  "${ModID}:Soulstone": {
+    "NAME": "Soulstone",
+    "DESCRIPTION": "Deal !D! damage. NL Startup: Gain !M! Gold. NL Exhaust."
+  },
+  "${ModID}:HiringSpecialists": {
+    "NAME": "Hiring Specialists",
+    "DESCRIPTION": "Gain !B! Block. NL Draw a card. NL Gain 1 ${ModID}:Hired_Support."
+  },
+  "${ModID}:Forcefield": {
+    "NAME": "Forcefield",
+    "DESCRIPTION": "Gain !B! Block X times. If you use this card having at least 3 [E] , gain 1 Blur.",
```

Comment:
The check for getting Blur counts the +2 you get from Chemical X, right? So playing this card with 1 energy and Chemical X would give you the Blur effect.

If that's right, it would be better to word this as "If X is 3 or more". That should also be shorter, which is nice.